acq4/devices/MicroManagerCamera/mmcamera.py

```diff
@@ -206,6 +207,38 @@ def _readAllParams(self):
 
             self._allParams = params
 
+    def getROI(self):
+        cam_region = self.mmc.getROI(self.camName)
```

Comment:
I know it hurts, but we should prolly `camRegion` for consistency.

Reply:
Oh man; I didn't even notice I did that. Yep.